modules/ROOT/pages/managing-schedules.adoc

```diff
@@ -4,33 +4,39 @@ include::_attributes.adoc[]
 endif::[]
 :keywords: schedules, cloudhub, polling
 
-The CloudHub console includes a *Schedules* tab in the left navigation menu where you can view and control any polling elements within the flows of your deployed applications. CloudHub automatically detects the polling elements from within your flows and lists them on the *Schedules* tab. On this tab, you can select the polling elements and enable or disable them, change the polling frequency, or run them immediately – all during runtime, without making any changes in the underlying application.
+The *Schedules* tab in the left navigation menu of the CloudHub console enables you to view and control any polling elements within the flows of your deployed applications. On this tab, you can manage polling elements, change the polling frequency, or run them immediately – all during runtime, without making any changes in the underlying application.
 
 == Use Cases
 
 The *Schedules* tab is useful in the following scenarios:
 
-* You have a job that runs nightly but you want to trigger it right now so you can debug it. You want to do this in runtime so you don’t need to update the schedule in your underlying application.
+* You have a job that runs nightly but you want to debug it now.
++
+You can trigger it while it's running without updating the schedule in your underlying application.
 * You want to change the schedules in your application at runtime without redeploying.
 * You want to disable a scheduled job while one of the applications you are connecting to is undergoing maintenance.
 * You want runtime visibility into the schedules in your application.
 
 == Considerations
```

Comment:
Be careful using this title to label a bunch of loosely related bits of information. This list looks pretty good. Sometimes writers use "Requirements for..." or "Restrictions to..." rather than "Considerations," which is a bit vague.